spyder/app/mainwindow.py

```diff
@@ -1302,14 +1302,6 @@ def post_visible_setup(self):
         for widget in self.floating_dockwidgets:
             widget.setFloating(True)
 
-        # In MacOS X 10.7 our app is not displayed after initialized (I don't

```

Comment:
@mrclary why is this no longer needed?

Reply:
I don't know what this code did in previous standalone apps in 10.7, but all it currently does is open Spyder.app/Contents/Resources/Spyder.py in the Editor when Spyder is launched...not desireable.

Reply:
No problem with removing this. It was a very old hack and a lot has changed since.